Examples do not run

Per the instructions on the home page, I did

```
$ git clone https://github.com/AllenFang/react-bootstrap-table.git
$ cd react-bootstrap-table
$ npm install
$ gulp example-server #after start, open browser and go to http://localhost:3004
```

This gave

```
[11:53:02] Starting 'example-server'...
[11:53:02] Finished 'example-server' after 60 ms
Listening at localhost:3004
webpack: wait until bundle finished: /
Time: 23398ms
        Asset     Size  Chunks       Chunk Names
app.bundle.js  7.35 MB       0       app
   vendors.js   663 kB       1       vendors
chunk    {0} app.bundle.js (app) 3.01 MB {1} [rendered]
chunk    {1} vendors.js (vendors) 224 kB [rendered]

ERROR in ./~/react-router/lib/Router.js
Module not found: Error: Cannot resolve module 'history/lib/createHashHistory' in <path>/react-bootstrap-table/node_modules/react-router/lib
 @ ./~/react-router/lib/Router.js 23:35-75

ERROR in ./~/react-router/lib/useRoutes.js
Module not found: Error: Cannot resolve module 'history/lib/Actions' in <path>/react-bootstrap-table/node_modules/react-router/lib
 @ ./~/react-router/lib/useRoutes.js 15:25-55

ERROR in ./~/react-router/lib/useRoutes.js
Module not found: Error: Cannot resolve module 'history/lib/useQueries' in <path>/react-bootstrap-table/node_modules/react-router/lib
 @ ./~/react-router/lib/useRoutes.js 17:28-61

ERROR in ./~/react-router/lib/match.js
Module not found: Error: Cannot resolve module 'history/lib/createMemoryHistory' in <path>/react-bootstrap-table/node_modules/react-router/lib
 @ ./~/react-router/lib/match.js 13:37-79

ERROR in ./~/react-router/lib/match.js
Module not found: Error: Cannot resolve module 'history/lib/useBasename' in <path>/eact-bootstrap-table/node_modules/react-router/lib
 @ ./~/react-router/lib/match.js 17:29-63
webpack: bundle is now VALID.
```

When I opened localhost:3004, it showed a blank page.

This is probably related to this message when running `npm install`:

```
├── UNMET PEER DEPENDENCY history@^1.17.0
```

Using Safari 10.0 on MacOS 10.10.5.

--- a/package.json
+++ b/package.json
@@ -64,6 +64,7 @@
   },
   "dependencies": {
     "classnames": "^2.1.2",
+    "history": "^1.17.0",
     "react-toastr": "^2.8.0"
   },
   "peerDependencies": {

--- a/package.json
+++ b/package.json
@@ -48,6 +48,7 @@
     "gulp-concat-css": "^2.2.0",
     "gulp-cssmin": "^0.1.7",
     "gulp-shell": "^0.5.2",
+    "history": "^1.17.0",
     "jquery": "^2.1.4",
     "jsx-loader": "^0.13.2",
     "react": "^0.14.3 || ^15.0.0",
@@ -64,7 +65,6 @@
   },
   "dependencies": {
     "classnames": "^2.1.2",
-    "history": "^1.17.0",
     "react-toastr": "^2.8.0"
   },
   "peerDependencies": {

